README.md

```diff
@@ -5,57 +5,35 @@
 
 ## How to Build
 
-### Visual Studio
+### Necessary Tools
 
-#### Necessary Tools
+* `dotnet` command line tool
```

Comment:
Should we add a link to it or just saying "`dotnet` command line tool" is sufficient for .NET developers to understand?